<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogVocabolario</class>
 <widget class="QDialog" name="DialogVocabolario">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Termini</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabConsulta">
      <attribute name="title">
       <string>Consultta Termini</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGestisci">
      <attribute name="title">
       <string>Gestisci Termini</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelImgTxt">
         <property name="text">
          <string>Immagine</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" rowspan="2">
        <widget class="QLabel" name="labelTermine">
         <property name="text">
          <string>Termine:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="5" colspan="3">
        <widget class="QLabel" name="labelImg">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Definizione:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item row="5" column="4" rowspan="2">
        <widget class="QListView" name="listView"/>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pushButtonNuovo">
         <property name="text">
          <string>Nuovo</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButtonElimina">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Elimina</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="pushButtonImg">
         <property name="text">
          <string>Immagine</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QPushButton" name="pushButtonSave">
         <property name="text">
          <string>Salva</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonNuovo</sender>
   <signal>clicked()</signal>
   <receiver>DialogVocabolario</receiver>
   <slot>nuovo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonElimina</sender>
   <signal>clicked()</signal>
   <receiver>DialogVocabolario</receiver>
   <slot>elimina()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonImg</sender>
   <signal>clicked()</signal>
   <receiver>DialogVocabolario</receiver>
   <slot>immagine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>DialogVocabolario</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>DialogVocabolario</receiver>
   <slot>select(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>select(QModelIndex)</slot>
  <slot>elimina()</slot>
  <slot>immagine()</slot>
  <slot>save()</slot>
 </slots>
</ui>
